lms/static/js/commerce/views/receipt_view.js

```diff
@@ -19,11 +19,6 @@ var edx = edx || {};
             this.useEcommerceApi = this.ecommerceBasketId || this.ecommerceOrderNumber;
             _.bindAll(this, 'renderReceipt', 'renderError', 'getProviderData', 'renderProvider', 'getCourseData');
 
-            /* Mix non-conflicting functions from underscore.string (all but include, contains, and reverse) into
-             * the Underscore namespace.
-             */
-            _.mixin(_s.exports());
```

Comment:
@andy-armstrong Here we have removed `_.str` usage and also instead of using `_.sprintf` which is method of `_.str` we have manually formatted the urls. So do you think it is a good enough solution for now or would you like to suggest some other solution.
FYI: @bjacobel, @symbolist